cli/src/template/ios/MoproApp/ContentView.swift

```diff
@@ -78,15 +78,18 @@ extension ContentView {
             let c = a*b
             inputs["a"] = [String(a)]
             inputs["b"] = [String(b)]
-            
+            let input_str: String = 
+                (try? JSONSerialization.data(withJSONObject: inputs, options: .prettyPrinted))
+                .flatMap { String(data: $0, encoding: .utf8) } ?? ""
```

Comment:
I think here can just be a string input 
e.g. `{"a": "3", "b": "5"}`, etc.

Reply:
fixed in[35305a1](https://github.com/zkmopro/mopro/pull/343/commits/35305a125cf40fff17b1e3c2d640ce5032c79f8d)